src/ResourceManager/RecoveryServices/Commands.RecoveryServices/Az.RecoveryServices.psd1

```diff
@@ -54,7 +54,8 @@ DotNetFrameworkVersion = '4.7.2'
 RequiredModules = @(@{ModuleName = 'Az.Profile'; ModuleVersion = '0.3.0'; })
 
 # Assemblies that must be loaded prior to importing this module
-RequiredAssemblies = '.\Microsoft.Azure.Commands.RecoveryServices.Backup.Models.dll', 
+RequiredAssemblies = '.\AutoMapper.dll', 
```

Comment:
Please add this into Az once the appropriate changes have been made

Reply:
Will do.